lib/screens/settings/settings_account.dart

```diff
@@ -1,22 +1,60 @@
+import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
+import 'package:project_eureka_flutter/screens/account_settings_screens/account_settings_delete.dart';
+import 'package:project_eureka_flutter/screens/account_settings_screens/account_settings_email.dart';
+import 'package:project_eureka_flutter/screens/account_settings_screens/account_settings_password.dart';
```

Comment:
Move folder `account_settings_screens` to `settings` folder and name it `account`